05-services.md

```diff
@@ -541,11 +541,31 @@ empty or undefined.
 
 #### Env_file format
 
-Each line in an `.env` file must be in `VAR[=[VAL]]` format. Lines beginning with `#` are ignored.
-Blank lines are also ignored.
-
-The value of `VAL` is used as a raw string and not modified at all. If the value is surrounded by quotes, as is often the case for shell variables, the quotes are included in the value passed to containers
```

Comment:
As this issue highlights https://github.com/compose-spec/compose-spec/issues/397, this paragraph is wrong. I've replaced with content taken from docs https://docs.docker.com/compose/environment-variables/env-file/